### ISSUE TITLE
[Add-on] SiliconLabs Zigbee/OpenThread Multiprotocol

### ISSUE THREAD

**Issue by mrninhvn** [lmahmutov/SGW-Homeassistant#39]
[https://github.com/home-assistant/addons-development/tree/master/silabs-multiprotocol](url)
Is add-on integration possible?

**Comment by lmahmutov** [comment]
Yes, it is possible. But we need more time.If you can help with this it would be very good
